packages/plugin-javascript/index.js

```diff
@@ -59,17 +59,32 @@ export function javascriptFile({ path, target }) {
   });
 }
 
+function isURLImport(target) {
+  try {
+    const { protocol, pathname } = new URL(target);
+    if (protocol !== 'https:' && protocol !== 'http:') return false;
+    const ext = extname(pathname);
+    if (ext !== '.js' && ext !== '.ts') return false;
+    return true;
+  } catch (err) {
+    return false;
+  }
+}
```

Comment:
Maybe we can simplify this by just checking if `target` is a valid url. It's not 100% inline with the spec of es modules , but that way it would work for similar cases as well. What do you think?

Reply:
I think checking for `:http(s)` as the protocol and `.{js,ts}` as the extension is pretty much the bare minimum of checks that need / should be done here. For example, `new URL("a:b")` is a valid URL (protocol `a:`, path `b`).